docs/nx-cloud/set-up/github.md

```diff
@@ -4,11 +4,37 @@
 
 The [Nx Cloud Github App](https://github.com/marketplace/official-nx-cloud-app) lets you access the result of every run—with all its logs and build insights—straight from your PR.
 
-## Nx Public Cloud CI Setup
+## Connecting Your Workspace
 
-Before you install the app, please make sure you have a **valid Nx Cloud `accessToken` in your `nx.json` file**. Nx Cloud reports will not generate properly on your PRs without one. You should already have one if you’re using Nx Cloud, but if you don’t, you can learn how to add one [here](/nx-cloud/account/users).
+Once you have installed the Nx Cloud Github App, you must link your workspace to the installation. To do this, sign in to Nx Cloud and navigate to the VCS Integrations setup page. This page can be found by clicking on the status card in the sidebar, or by clicking "Workspace Options" -> "VCS Integrations".
 
-You can find and install the app from [the GitHub marketplace](https://github.com/marketplace/official-nx-cloud-app).
+![Access VCS Setup](/nx-cloud/set-up/access-vcs-setup.png)
+
+Once on the VCS Integrations setup page, select "Github" in the platforms dropdown. You will be prompted to enter the name of your repository.
+
+![Add Github Repository](/nx-cloud/set-up/add-github-repository.png)
+
+### Choosing an Authentication Method
+
+It is easier to configure the Nx Cloud Github Integration to use its Github App to authenticate, and this method should be preferred for users on Nx Public Cloud. Advanced users, or Nx Enterprise clients may instead wish to generate a personal access token instead.
+
+#### Using the Github App
+
+To use the Nx Cloud Github App for authentication, select the radio button and then click "Test Connection". This will verify that Nx Cloud can connect to your repo. Upon a successful test, your configuration is saved. Check the CI Platform Considerations section below and if there are no additional instructions for your platform of choice, setup is complete.
```

Comment:
```suggestion
To use the Nx Cloud GitHub App for authentication, select the radio button and then click "Test Connection". This will verify that Nx Cloud can connect to your repo. Upon a successful test, your configuration is saved. Check the CI Platform Considerations section below and if there are no additional instructions for your platform of choice, setup is complete.
```